by-language/csharp-npgsql/tests/DemoProgramTest.cs

```diff
@@ -215,5 +212,21 @@ public async Task TestArrayPocoExample()
 
         }
 
+        [Fact]
+        public async Task TestGeoJsonTypesExample()
+        {
+            var conn = fixture.Db;
+
+            // Provision data.
+            var task = new DatabaseWorkloadsTypes(conn).GeoJsonTypesExample();
+            var point = await task.WaitAsync(TimeSpan.FromSeconds(0.5));
+
+            // Validate the outcome.
+            var coords = new Point(new Position(85.43, 66.23)).Coordinates;
+            Assert.Equal(coords.Latitude, point?.Coordinates.Latitude);
+            Assert.Equal(coords.Longitude, point?.Coordinates.Longitude);
```

Comment:
Can't directly compare GeoJSON object instances. Need to run the comparison on the individual coordinates inside. 🤷